ModelBundle/Document/Site.php

```diff
@@ -318,4 +318,23 @@ public function getMainAliasId()
     {
         return $this->aliases->indexOf($this->getMainAlias());
     }
+
+    /**
+     * Return alias id for given language
+     *
+     * @param string $language
+     *
+     * @return int
+     */
+    public function getAliasIdForLanguage($language)
```

Comment:
What is the aim of this function ? If I refer to its name, it should return an aliasId for the combination of a site/a language. But a site can have several aliases with the same language.
Is there a problem with that ?